cogs/config.py

```diff
@@ -90,6 +94,10 @@
                                         config.get(SEC_REPMOD_MOODLE, 'USERNAME'))
 REPMOD_MOODLE_PASSWORD = os.environ.get('COGS_REPMOD_MOODLE_PASSWORD',
                                         config.get(SEC_REPMOD_MOODLE, 'PASSWORD'))
+AUTHMOD_LDAP_HOST = os.environ.get('COGS_AUTHMOD_LDAP_HOST',
+					config.get(SEC_AUTHMOD_LDAP, 'HOST'))
```

Comment:
These should be spaces, not tabs. Might be good to just run a tab-to-spaces converter on all of the files since it looks like Dirk was editing with tabs.